doc/source/faqs.rst

```diff
@@ -8,7 +8,19 @@ Nextflow
 If Nextflow crashes, where do I look for the error message?
 -----------------------------------------------------------
 
-When Nextflow crashes, an error message appears on the screen indicating which process exited the pipeline and reporting the error code. More details about the error that stopped the pipeline are stored in log files accessible in ``rate_adjustment/<focal_species>/log_XXXXXXXX``. The log folder name is unique for each run and is reported on the terminal at the beginning of a Nextflow run.
+When Nextflow crashes, an error box appears in the terminal and shows which process stopped the pipeline, the error message lines and points to complete output files. For example:
+
+.. figure:: _images/error_box1.png
+    :align: center
+    :width: 800
+
+    Here the error is traced back to ``setupAdjustment`` process and it is due to a wrong setting in the *ksrates* configuration file concerning the input phylogeny.
+
+.. figure:: _images/error_box2.png
```

Comment:
Make sure these images correspond to the actual output from the latest code.

Reply:
Done.